static/js/lib/ckeditor/plugins/CustomLink.ts

```diff
@@ -23,18 +51,56 @@ class CustomLinkCommand extends LinkCommand {
       }
     }
 
-    getExternalResource(this.editor.config.get(WEBSITE_NAME), href, title).then(
-      (externalResource) => {
+    if (syntax.isResourceLinkHref(href)) {
+      super.execute(href)
+    } else {
+      getExternalResource(
+        this.editor.config.get(WEBSITE_NAME),
+        href,
+        title,
+      ).then((externalResource) => {
         if (externalResource) {
           updateHref(externalResource, this.editor, (href) =>
             super.execute(href),
           )
         }
-      },
-    )
+      })
+    }
   }
 }
 
+/**
+ * CustomLink Plugin for CKEditor 5
+ *
+ * PURPOSE:
+ * This plugin automatically converts external URLs into tracked "external resources"
```

Comment:
```suggestion
 * This plugin automatically converts URLs into tracked "external resources"
```

My understanding is...

All URLs are "external resources", where "external" means eternal to the current site. (A URL within the current site would technically be handled as an external resource, too, but the expectation is that site authors use resource links directly for that sort of thing.)